package.json

```diff
@@ -7,6 +7,10 @@
   "repository": {
     "url": ""
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/felixge/node-urun.git"
+  },
```

Comment:
This duplicates the `"repository"` key from above which renders the `package.json` invalid or at least of questionable interoperability.